.dockerfiles/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+wait_for() {
+	host="${1}"
+	port="${2}"
+	name="${3}"
+	timeout="${4:-30}"
+
+	printf "Waiting for %s on %s:%s ... " "${name}" "${host}" "${port}"
+
+	i=0
+	while ! nc -z "${host}" "${port}"; do
+		sleep 1
+		i=$((i + 1))
+		if [ "$i" -gt "$timeout" ]; then
+			printf "Timed out!\n"
+			return 1
+		fi
+	done
+	printf "OK\n"
+}
+
+netdata -D >netdata.log 2>&1 &
```

Comment:
Depending on the distro, Netdata may actually already have been started. We should account for that here.

Reply:
Hmm actually I'm not certain about this myself. In many cases the Docker image for many distros doesn't actually start the typical "init" process. In the case of SystemD this means even though the package installed the unit file, it won't be started.

This is okay because we're not really interested in testing SystemD here -- We should of course (_separately_) ensure we properly test our SystemD (_or other startup mechanisms_) elsewhere.

Reply:
Yeah, systemd won't start it, but I'm not sure that 'conventional' init systems won't. Debian's old-style init.d stuff might in this case (though I don't think any of that is being used in the Debian Docker images.